db-cleanup/airflow-db-cleanup.py

```diff
@@ -194,65 +239,71 @@ def cleanup_function(**context):
 
-        if keep_last_group_by is not None:
-            subquery = subquery.group_by(keep_last_group_by)
-            logging.info("SUB QUERY [keep_last_group_by]: " + str(subquery))
+            if keep_last_group_by is not None:
+                subquery = subquery.group_by(keep_last_group_by)
+                logging.info(
+                    "SUB QUERY [keep_last_group_by]: " + str(subquery))
 
-        subquery = subquery.from_self()
+            subquery = subquery.from_self()
 
-        query = query.filter(
-            and_(age_check_column.notin_(subquery)),
-            and_(age_check_column <= max_date)
-        )
+            query = query.filter(
+                and_(age_check_column.notin_(subquery)),
+                and_(age_check_column <= max_date)
+            )
 
-    else:
-        query = query.filter(age_check_column <= max_date,)
+        else:
+            query = query.filter(age_check_column <= max_date,)
 
-    entries_to_delete = query.all()
+        entries_to_delete = query.all()
 
-    logging.info("Query: " + str(query))
-    logging.info(
-        "Process will be Deleting the following " +
-        str(airflow_db_model.__name__) + "(s):"
-    )
-    for entry in entries_to_delete:
+        logging.info("Query: " + str(query))
         logging.info(
-            "\tEntry: " + str(entry) + ", Date: " +
-            str(entry.__dict__[str(age_check_column).split(".")[1]])
+            "Process will be Deleting the following " +
+            str(airflow_db_model.__name__) + "(s):"
         )
+        for entry in entries_to_delete:
+            logging.info(
+                "\tEntry: " + str(entry) + ", Date: " +
+                str(entry.__dict__[str(age_check_column).split(".")[1]])
+            )
 
-    logging.info(
-        "Process will be Deleting " + str(len(entries_to_delete)) + " " +
-        str(airflow_db_model.__name__) + "(s)"
-    )
+        logging.info(
+            "Process will be Deleting " + str(len(entries_to_delete)) + " " +
+            str(airflow_db_model.__name__) + "(s)"
+        )
+
+        if ENABLE_DELETE:
+            logging.info("Performing Delete...")
+            # using bulk delete
+            query.delete(synchronize_session=False)
+            session.commit()
+            logging.info("Finished Performing Delete")
+        else:
+            logging.warn("You're opted to skip deleting the db entries!!!")
 
-    if ENABLE_DELETE:
-        logging.info("Performing Delete...")
-        # using bulk delete
-        query.delete(synchronize_session=False)
-        session.commit()
-        logging.info("Finished Performing Delete")
-    else:
-        logging.warn("You're opted to skip deleting the db entries!!!")
+        logging.info("Finished Running Cleanup Process")
 
-    logging.info("Finished Running Cleanup Process")
+    except ProgrammingError:
+        print(str(airflow_db_model) +
+              " is not present in the metadata. Skipping...")
```

Comment:
Include the full exception message from the ProgrammingError exception for debug purposes